### DISCUSSION TITLE
Highlighting pointing centers

### DISCUSSION THREAD

**Discussion by vpassegger** [Subaru-PFS/spt_target_uploader#179] [Ideas]
I have an idea for the plots showing target distribution and pointing centers in the Uploader. It would be nice to be able to click on the pointing centers (grey squares) in the plot and then only those targets being observed in this pointing are highlighted. The PPC_id from the top plot needs to be added here (what is PA:0 that is shown now?). I know that PPCs will ultimately change during scheduling considering fiber sharing, but it would be a nice feature for the user to get a better feeling of when/how the targets are observed. 

One more thing: In the User Guide, it would be helpful to specify that 'pointing centers' refers to 'telescope pointing centers'. I was confused at first, because I thought it means fiber pointings and in that case the calculated numbers in the Uploader don't make sense. Maybe other users could have the same problem.

**Comment by monodera** [comment]
Thanks for the comment. The function highlighting objects in a selected pointing would be a nice and reasonable function to add. Let's see if I can manage to implement it (I'm still not quite familiar with these interactive plotting with callbacks). 

I'll modify the documentation as you suggested to be explicit on what 'pointing centers' mean.
